Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -919,11 +924,60 @@ def change_attack_pattern_to_stix_attack_pattern(indicator: dict[str, Any]):
 
         return indicator
 
+    @staticmethod
+    def get_entity_b_type_and_value(related_obj: str, id_to_object: dict[str, dict[str, Any]],
+                                    is_unit42_report: bool = False) -> tuple:
+        """
+        Gets the type and the value of the indicator.
```

Comment:
```suggestion
        Gets the type and the value of the indicator.
```
 Gets the type and value of the indicator in entity_b.